client/components/PetTypeTile.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+const PetTypeTile = props => {
+  let imgURL = props.url
```

Comment:
insead of conditionally applying the image source, you might consider conditionally rendering the entire image element - then if there is not one present, you could display some text like `No image available`